src/proc/spatial-filter.cpp

```diff
@@ -61,8 +61,8 @@ namespace librealsense
         _holes_filling_mode(holes_fill_def),
         _holes_filling_radius(0)
     {
-        _stream_filter = RS2_STREAM_DEPTH;
-        _stream_format_filter = RS2_FORMAT_Z16;
+        _stream_filter.stream = RS2_STREAM_DEPTH;
```

Comment:
Better to be move to initialization list, so if one day one other class member become dependent on this struct, the initialization order will be preserved, or at least compiler warning will be generated